### ISSUE TITLE
Twitter Support

### ISSUE THREAD

**Issue by gregseed** [dialogflow/dialogflow-fulfillment-nodejs#23]
Is this ever going to be included, or has this been deprecated? (I know there is a lot of flux right now with the Account Activity API)

**Comment by matthewayne** [comment]
Twitter, unfortunately, has not ever been supported by this library with the native responses (text, card, images, etc.), due to Dialogflow not supporting message objects for Twitter but you can create a custom payload for Twitter responses: https://github.com/dialogflow/dialogflow-fulfillment-nodejs/blob/master/docs/Payload.md

I'll consider this a feature request and leave this open for further comments.

**Comment by Ashwin-Kapes** [comment]
I've added two Text Responses in DialogFlow Welcome Intent but in twitter DM I'm getting any one response that is either Line one or Line Two. why so? Is it possible to use JSON Payload instead of Text Response when connected to twitter.

![response](https://user-images.githubusercontent.com/510118/41774493-9d5095f2-763d-11e8-9ad2-54ba2ca7f027.png)

**Comment by gouthamgtm** [comment]
> Twitter, unfortunately, has not ever been supported by this library with the native responses (text, card, images, etc.), due to Dialogflow not supporting message objects for Twitter but you can create a custom payload for Twitter responses: https://github.com/dialogflow/dialogflow-fulfillment-nodejs/blob/master/docs/Payload.md
> 
> I'll consider this a feature request and leave this open for further comments.

The link you have provided is broken. It would be great if you could provide a sample custom payload where we can send rich messages like Twitter's Quick Replies.

**Comment by sarahdwyer** [comment]
Yes certainly, here's the reference to our [Twitter-related Dialogflow docs](https://dialogflow.com/docs/integrations/twitter) and below is an example of a payload response:

```
const googlePayloadJson = { 
  expectUserResponse: true,
  isSsml: false,
  noInputPrompts: [],
  richResponse: {
    items: [{ simpleResponse: { textToSpeech: 'hello', displayText: 'hi' } }]
  },
  systemIntent: {
    intent: 'actions.intent.OPTION',
  }
}

let payloadResponse = new PayloadResponse(PLATFORMS.ACTIONS_ON_GOOGLE, {});
payloadResponse.setPayload(googlePayloadJson);
```

PayloadResponse is a constructor that extends from the RichResponse class. To read more about the RichResponse class and the message object in the [Dialogflow(v2) docs](https://dialogflow.com/docs/reference/api-v2/rest/v2/projects.agent.intents).

**Comment by VamsidharMuggulla** [comment]
Why this is still open!!??

**Comment by gouthamgtm** [comment]
You can close this
